streams-data/solutions/vehicles/scripts/produce_vehicle_positions.py

```diff
@@ -0,0 +1,65 @@
+"""
```

Comment:
This script lies here since it is sth directly relevant with the app right?

Even if this is sth related with the app would you consider to move it to solutions/scripts (same for the apps)?

I mean we are keepiing our app related yamls in apps/resources or solutions/resources as well, so IMO it can be the same for app related scripts. WDYT?

Reply:
In this case, I just wanted to reflect the same folder structure we had in the `apps` folder